### ISSUE TITLE
MVIEWS and Temporary Tables "WARNING: can not extract data from <MVIEW NAME>, no column found..."

### ISSUE THREAD

**Issue by jagmohankaintura-tl** [darold/ora2pg#966]
Oracle Schema being extracted contains Mviews, Temp tables and Normal tables. While extracting for Mviews and Temporary tables gets a Warning Message "no column found" . Once this error arising the next good table data is merged into the file created for Mview. 

Example while extraction here are tables in Order:
A_SNAP : Mview
B_SNAP: Mview
CBC : Normal Table
DFG : Normal Table

While extracting first Mview and Temporary tables are being considered for extraction and during extraction file being creating is A_SNAP_data.sql , it contains the Statements as 👍 

in File **"A_SNAP_data.sql"**
DELETE FROM A_SNAP where 1=1;
DELETE FROM B_SNAP where 1=1;
DELETE FROM  CBC where 1=1;
\copy for table CBC.

So all above is getting merged in single file because of ERROR and file for CBC_data.sql is not being created.
I think while extraction if error or WARNING comes we should move for extraction of next table and created seperate files.
Another thing we can ignore extraction of data for MVIEWS and Temporary tables.
[BERTH_SNAP_data.txt](https://github.com/darold/ora2pg/files/5212582/BERTH_SNAP_data.txt)

In attached file the good table is bis_cargo_bill_svc_event.

**Comment by jagmohankaintura-tl** [comment]
Hi @darold  Can you please guide for this error . We have many Mviews and Temporary tables in Oracle database.

**Comment by darold** [comment]
Commits b021ca and 627e60d remove attempt to dump data from MVIEWs which leads to "no column found" warning and prevent dumping internal indexes from MVIEWs. This issue might be fixed.

**Comment by jagmohankaintura-tl** [comment]
Thanks alot @darold . I will test and will share Result.
